src/core/tsi/test_creds/README

```diff
@@ -99,6 +99,25 @@ multi-domain-openssl.cnf
 $ openssl req -x509 -new -extensions v3_req -key multi-domain.key -out
 multi-domain.pem -days 3650 -config multi-domain-openssl.cnf
 
+
+Intermediate is an intermediate CA used to sign leaf certs used to test having
```

Comment:
nit: This comment is a bit circular, could you clarify? Also please mention the other files (e.g. leaf_and_intermediate_chain.pem) that are being created.

Reply:
Done